ePBFT/src/pbft/libbyz/Certificate.h

```diff
@@ -176,6 +176,8 @@ class Certificate
   T* mym; // my message in this or null if I have no message in this
   Time t_sent; // time at which mym was last sent
 
+  uint32_t f; // the value of f when starting to run
```

Comment:
I would add a comment here along these lines: "Note that is is necessary that f stays constant for the lifetime of the certificate since a request may change the value of f".

Reply:
i changed it to be const which I believe implies the same thing